apps/ergw_core/src/ergw_gtp_c_socket.erl

```diff
@@ -174,8 +174,8 @@ init({Name, #{ip := IP, vrf := VRF,
 	       v1_seq_no = 0,
 	       v2_seq_no = 0,
 	       pending = gb_trees:empty(),
-	       requests = ergw_cache:new(?T3 * 4, requests),
-	       responses = ergw_cache:new(?CACHE_TIMEOUT, responses),
+	       requests = ergw_cache:new(100, requests),
```

Comment:
Do we need also set `100` into [ergw_sx_socket.erl#L168](https://github.com/travelping/ergw/blob/master/apps/ergw_core/src/ergw_sx_socket.erl#L168)?